python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -881,12 +884,14 @@ def get_updated_parent_asset_partitions(
             )
         return updated_parents
 
-    def get_root_unreconciled_ancestors(
+    def get_outdated_ancestors(
```

Comment:
"root" threw me off here, because normally "root" means assets at the root of the graph. Ideally there would be a term that communicates the fact that "If two ancestors would be returned, but one of them is an ancestor of the other one, then only the most upstream ancestor is included." But I had trouble coming up with one.